target/scripts/startup/setup-stack.sh

```diff
@@ -960,6 +960,22 @@ function _setup_security_stack
       sed -i "s|\$final_spam_destiny.*=.*$|\$final_spam_destiny = D_BOUNCE;|g" /etc/amavis/conf.d/49-docker-mailserver
       sed -i "s|\$final_bad_header_destiny.*=.*$|\$final_bad_header_destiny = D_BOUNCE;|g" /etc/amavis/conf.d/49-docker-mailserver
     fi
+
+    if [[ ${ENABLE_SPAMASSASSIN_KAM} -eq 1 ]]
+    then
+      _notify 'inf' 'Configuring Spamassassin KAM'
+      local SPAMASSASSIN_KAM_CRON_FILE=/etc/cron.daily/10-spamassassin_kam
+
+      sa-update --import /etc/spamassassin/kam/kam.sa-channels.mcgrail.com.key
+      cat >"${SPAMASSASSIN_KAM_CRON_FILE}" <<"EOM"
+#! /bin/bash
+
+sa-update --gpgkey 24C063D8 --channel kam.sa-channels.mcgrail.com -v 2>&1
```

Comment:
```suggestion
sa-update --gpgkey 24C063D8 --channel kam.sa-channels.mcgrail.com 2>&1
```

Otherwise there will be output (an email) every day the cronjob runs.

Reply:
Would it be better to just extract this to it's own script file in a `target/cron/` or similar location? (_we also have `/etc/dms-settings` (file) and `/etc/dms` (directory)_)

I know it's small, but it would seem a little cleaner to centralize anything cron specific? Or do you feel inline generation like this is better suited? Personally the syntax and indentation to support it vs copying a file over doesn't seem like a great tradeoff?

We have a few different ways in this file alone for cron jobs. Multiple echo calls (1 per line for written file), `mv` for third-party and `sed` (adjustment fix), `chmod` differing permissions across the cron jobs too. That could be addressed by a separate PR, just raising awareness that this approach muddies the already inconsistent handling.

Reply:
Probably a good idea, handling cron in a separate function maybe. I will keep this in mind and when I come to it, provide a PR that takes care of uniform `cron`-handling.

Reply:
What I do in other projects is, to build the exact directory structure under "rootfs" and put the files there:

```
rootfs/etc/cron
rootfs/etc/supervisord
rootfs/bin
```

That way you have a clear overview about the files and how they will be in the image + only one COPY instruction is needed in Dockerfile --> `COPY rootfs /`